packages/bindgen/spec.yml

```diff
@@ -632,6 +637,9 @@ classes:
       # This is added due to DescriptorOrdering not being exposed
       get_results_description: '(results: const Results&) -> StringData'
       feed_buffer: '(ws: WatchStream&, buffer: BinaryData)'
+      # Converts char* and length to a combined string_view argument to callback.
+      # TODO: Consider making preverify_ok a bool.
```

Comment:
I really dislike the name `preverify_ok`, but I think that's out of our hands, or?

Reply:
Yeah that's for Core 👍